api/src/data/models/data-ingestion-mapping-model.ts

```diff
@@ -0,0 +1,8 @@
+export interface DataIngestionMapping {
+  id?: number;
+  source_id: number;
+  source_attribute: string;
+  source_value: string | null | undefined;
```

Comment:
This style does not match the style used to define the ID column. Prefer setting things as optional only one way in a given type (and preferably the whole codebase)